eli5/sklearn/explain_prediction.py

```diff
@@ -371,9 +398,26 @@ def explain_prediction_tree_regressor(
         target_names=None,
         targets=None,
         feature_names=None,
+        feature_re=None,
+        feature_filter=None,
         vectorized=False):
     """ Explain prediction of a tree regressor.
 
+    See :func:`eli5.explain_prediction` for description of
+    ``top``, ``target_names``, ``targets``, ``feature_names``,
+    ``feature_re`` and ``feature_filter`` parameters.
+
+    ``vec`` is a vectorizer instance used to transform
+    raw features to the input of the classifier ``clf``
```

Comment:
Must be some GH bug, I bet the comment was for the correct line (this https://github.com/TeamHG-Memex/eli5/pull/132/files/ea0c16abdd1629890ed97dd352d623c20e33de98#diff-5a5dfe2e408016505008ee23671cb61fR411), but now it's show in the wrong place for me.
Fixed in dec5f20. Thanks for taking care of the docs, I didn't check them careful enough.